CoreOnly/Sources/Firebase.h

```diff
@@ -99,12 +99,6 @@ FirebaseAnalytics dependency to your project to ensure Messaging works as intend
 
   #if __has_include(<FirebasePerformance/FirebasePerformance.h>)
     #import <FirebasePerformance/FirebasePerformance.h>
```

Comment:
Do we need to keep this? What is the purpose of this?

Reply:
It's needed. That allows clients to access Perf APIs by including Firebase.h or importing the Firebase module.

Reply:
@ramanpreetSinghKhinda These are Umbrella headers. If the including app includes Firebase.h, all the header files of other Firebase products that used by the app will be automatically imported. This is a huge convenience header for the developers. So this is definitely needed.